servers/physics_2d/space_2d_sw.h

```diff
@@ -96,13 +96,13 @@ class Space2DSW {
 
 	Set<CollisionObject2DSW *> objects;
 
-	Area2DSW *area;
+	Area2DSW *area = nullptr;
 
-	real_t contact_recycle_radius;
-	real_t contact_max_separation;
-	real_t contact_max_allowed_penetration;
-	real_t constraint_bias;
-	real_t test_motion_min_contact_depth;
+	real_t contact_recycle_radius = 1.0;
+	real_t contact_max_separation = 1.5;
+	real_t contact_max_allowed_penetration = 0.0;
+	real_t constraint_bias = 0.0;
+	real_t test_motion_min_contact_depth = 0.0;
```

Comment:
These don't match the original constructor:
```suggestion
	real_t contact_max_allowed_penetration = 1.5;
	real_t constraint_bias = 0.3;
	real_t test_motion_min_contact_depth = 0.005;
```